### ISSUE TITLE
Videos pause unprompted

### ISSUE THREAD

**Issue by artnathanh** [eramdam/BetterTweetDeck#588]
**Describe the bug**
Any video that is watched pauses somewhere early in the video. Cannot be worked around, you must open the video outside of tweetdeck. Tested with all extensions off but BTD, issue still occurs in newest update of Chrome.

**To Reproduce**
1. Play a Video
2. The video will stop

**Expected behavior**
The video should play, not stop

**Desktop (please complete the following information):**
 - OS: Windows 10 10.0.19042 Build 19042
 - Browser: Chrome
 - Version: 89.4389.114 (Official Build) (64-Bit)

**Comment by eramdam** [comment]
I've had a couple of reports about this but I have never been able to reproduce and I have no idea what part of my code would even trigger that 😕

Could you put links to the tweets on which the video pauses itself? Could you perhaps open the [web console](https://appuals.com/open-browser-console/) side by side and send me whatever appears (if anything) when the video pauses?

**Comment by eramdam** [comment]
@artnathanh also, can you upload your settings file (Settings -> Import/Export -> Download settings)? I wonder if a specific setting somehow triggers this

**Comment by artnathanh** [comment]
Absolutely - I'll try and get a console look today too. It isn't very consistent, definitely annoying. Sorry to hear about the bookmarks too, I should've looked further before posting (never use github)
settings -
[better-tweetdeck-settings-4.0.3-2021-04-13_13.03.54.txt](https://github.com/eramdam/BetterTweetDeck/files/6305434/better-tweetdeck-settings-4.0.3-2021-04-13_13.03.54.txt)

**Comment by artnathanh** [comment]
Here's some broken video examples. This first one wouldn't play more than 1 second https://twitter.com/monistreams/status/1384555824199225344
![chrome_3GgS4DJfxq](https://user-images.githubusercontent.com/30128101/115437641-6b4bbc00-a1da-11eb-8a71-7eee684005f4.png)

This one was fine https://twitter.com/POTUS/status/1384550246211104768

As far as I can tell, there are never any issues in the console. It looks the same across working videos and broken videos. Sometimes a refresh can fix it.

Let me know if there's anything I can do to help.

**Comment by YoderofKansas** [comment]
So I've been having this issue too, did some tests and I think we can rule out Better TweetDeck.  I tried disabling the extension and restarting Chrome, found a broken video.  Uninstalled BTD and restarted Chrome, found another broken video.  Both videos worked fine when I opened them on Twitter.  Might be a Chrome update or something behind the scenes at Twitter.

- Browser: Chrome Version 90.0.4430.93 (Official Build) (64-bit)
- OS: Windows 10, 64-bit, Ver. 10.0.19041, Build 19041.
